src/applications/static-pages/burials-v2-how-do-i-apply-widget/components/App/index.js

```diff
@@ -199,10 +199,9 @@ export const App = () => {
           If you don’t have their DD214 or other separation documents, you can
           request these documents now.
         </p>
-        <va-link
```

Comment:
Moving away from VADS components tends to be against best practice on the Platform. Have you registered an issue with the design system team about the defect occurring with the `va-link` component?

Reply:
Appreciate your comment @pjhill, I updated the logic to use va-link!

I was not able to test the static pages with my local build. I was able to make the same updates to va-link in devtools on production and verified the links should function correctly.